src/controller.coffee

```diff
@@ -1381,7 +1381,7 @@ Editor::getClosestDroppableBlock = (mainPoint) ->
     w: MAX_DROP_DISTANCE * 2
     h: MAX_DROP_DISTANCE * 2
   }, (point) =>
-    unless (point.acceptLevel is helper.DISCOURAGE) and not event.shiftKey
+    unless (point.acceptLevel is helper.DISCOURAGE) and not event?.shiftKey
```

Comment:
It still feels a little weird to me that getClosestDroppableBlock needs an event. What if we instead make this signature `Editor::getClosestDroppableBlock = (mainPoint, isDebugMode)` and then the caller is `@getClosestDroppableBlock(mainPoint, event?.shiftKey)`?